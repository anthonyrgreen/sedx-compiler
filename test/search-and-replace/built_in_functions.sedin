goodbye 1 hellohellohellohello 0 x These words should be moved to the front, followed by the number "1":
