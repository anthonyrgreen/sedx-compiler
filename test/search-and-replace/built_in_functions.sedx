match
${capture(maybe('goodbye')) as maybe_goodbye} ${capture(any()) as one_char} ${capture(star('hello')) as many_hello} ${maybe('non-existent ')}${capture(noneOf('a-z')) as noneof_a_z} ${capture(anyOf('a-z')) as anyof_a_z} ${capture(star(any())) as many_chars}
substitute
${many_chars} ${one_char} ${many_hello} ${maybe_goodbye} ${anyof_a_z} ${noneof_a_z}
