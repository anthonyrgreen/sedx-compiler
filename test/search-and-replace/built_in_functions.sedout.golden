These words should be moved to the front, followed by the number "1": 1 hellohellohellohello goodbye x 0
